### PR TITLE
add first 3 deep dive notebooks

### PR DESCRIPTION
**Pull request by andrewferlitsch**

### CODE REVIEW

File: notebooks/deepdive/custom/ucaip_customjob.ipynb [hunk 1]
```diff
@@ -0,0 +1,1857 @@
+{
```
Comment (thedriftofwords) [inline notebooks/deepdive/custom/ucaip_customjob.ipynb:1]: <p>The "Before you begin" section in cloud.google.com tutorials is reproduced in notebook form in the "Set up your GCP project" cell below, so you should delete this:</p><p> </p><h3>Follow before you begin in Guide</h3><p> </p><p>{ add link to any online before you begin tutorial on the product }</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/217/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (thedriftofwords) [inline notebooks/deepdive/custom/ucaip_customjob.ipynb:1]: <p>Looks like a cut-and-paste error happened here?</p><p> </p><p><strong>Note</strong>: Jupyter runs lines prefixed with&nbsp;<code>!</code>&nbsp;as shell commands, and it interpolates Python variables prefixed with `###&nbsp;Set up your GCP project </p><p><strong>The following steps are required, regardless of your notebook environment.</strong> </p><ol><li>into these commands.</li></ol><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/217/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (thedriftofwords) [inline notebooks/deepdive/custom/ucaip_customjob.ipynb:1]: <p>This sentence is outdated:</p><p> </p><p>"You can then&nbsp;create an AI Platform model version based on this output in order to serve&nbsp;online predictions."</p><p> </p><p>If this is in your notebook template for AI Platform (Unified) notebooks, please update that to replace "model version" with "endpoint", in the template and in all affected notebooks.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/217/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Comment (thedriftofwords) [inline notebooks/deepdive/custom/ucaip_customjob.ipynb:1]: <p>If users need to enable the Container Registry API, add that to the "Set up your GCP project" section within step 3, where they're enabling the AI Platform and Compute Engine APIs.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/217/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Comment (thedriftofwords) [inline notebooks/deepdive/custom/ucaip_customjob.ipynb:1]: <h4>"Import AIP AI Platform (Unified) SDK "  </h4><p> </p><p>Delete "AIP"?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/217/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Comment (thedriftofwords) [inline notebooks/deepdive/custom/ucaip_customjob.ipynb:1]: <p>FYI - The Stackdriver product name has changed to a suite of products called "Operations" (https://cloud.google.com/products/operations). The product suite includes: Cloud Logging, Cloud Monitoring, Cloud Trace, Cloud Debugger, and Cloud Profiler.</p><p> </p><p>Here, you'd use "Cloud Logging" - please make this replacement in all affected notebooks.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/217/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>

Comment (thedriftofwords) [inline notebooks/deepdive/custom/ucaip_customjob.ipynb:1]: <p>Don't use the internal link (pantheon.corp.google.com) in a public tutorial. Swap it for "console.cloud.google.com", and please update this throughout all affected notebooks.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/217/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>

Comment (thedriftofwords) [inline notebooks/deepdive/custom/ucaip_customjob.ipynb:1]: <p>"Google Storage" --&gt; "Cloud Storage" throughout.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/217/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='86'/>

Comment (thedriftofwords) [inline notebooks/deepdive/custom/ucaip_customjob.ipynb:1]: <p>Can you delete this "End of tutorial" cell? It's not necessary</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/217/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='102'/>

Comment (andrewferlitsch) [reply notebooks/deepdive/custom/ucaip_customjob.ipynb:1]: My cell is rendering correctly?

**Pull request closed without merging**